--- origsrc/gnuplot-4.2.6/src/Makefile.am	2010-01-23 22:51:05.120971800 -0600
+++ src/gnuplot-4.2.6/src/Makefile.am	2010-01-23 23:29:48.704873200 -0600
@@ -61,8 +61,7 @@ getcolor_x11.o: getcolor.c
 gnuplot_x11_LDADD = getcolor_x11.o $(XLIBS)
 
 if BUILD_LUA
-pkglibexec_PROGRAMS += gnuplot.lua
-gnuplot_lua_SOURCES =
+pkglibexec_SCRIPTS = gnuplot.lua
 gnuplot.lua: $(top_srcdir)/term/lua/gnuplot.lua
 	cp $(top_srcdir)/term/lua/gnuplot.lua .
 endif
